server/index.js

```diff
@@ -1,4 +1,4 @@
-require('dotenv').config(); //loading environment vars
+require('dotenv').config(); //loading environment lets
```

Comment:
Pro tip: This will work locally but might cause issues when it is deployed. You can pass a config object into the config() call to tell it where to find the .env file. On the EC2 instance it will look in the current directory and not the root, which caused issues for both me and Chris last week.

```
require('dotenv').config({
   path: "../.env"
});
```

Reply:
Thanks soooo much for heads up, will correct today.